gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py

```diff
@@ -3962,3 +3962,11 @@ def impl(context):
     cmd =  "sudo mv -f /tmp/hosts_orig /etc/hosts; rm -f /tmp/clusterConfigFile-1; rm -f /tmp/hostfile--1"
     context.execute_steps(u'''Then the user runs command "%s"''' % cmd)
 
+@given(u'"LANG" is different from English')
+def step_impl(context):
+	os.environ['LANG'] = 'ru_RU.utf8'
```

Comment:
what if ru_RU.utf8 not in available locales?
for example
```sh
LANG=qwe ls -lan diskquota*
ls: cannot access 'diskquota*': No such file or directory
LENG=qwe ls -lan diskquota*
ls: невозможно получить доступ к 'diskquota*': Нет такого файла или каталога
```

Reply:
It seems that existing check doesn't validate the problem.

1. I've reverted changes for gpMgmt/bin/gppylib/commands/base.py
    ```
    git checkout adb-6.x-dev -- gpMgmt/bin/gppylib/commands/base.py
    ```
3. Build Docker container
    ```
    docker build -f arenadata/Dockerfile .
    ```
5. Run gpstop tests inside
    ```
    export IMAGE=sha256:<hash>
    bash arenadata/scripts/run_behave_tests.bash gpstop
    ```

Reply:
`LANG` env variable didn't affect the language, so I switched back to `LC_ALL`. Also, the error is printed to stdout, so the test was always successful, I fixed that too

Reply:
I also added Russian locale generation, so there's shouldn't be a problem in a missing locale

Reply:
I try to reproduce error without patch as describe above, but got
```sh
sudo: locale-gen: command not found
1 feature passed, 0 failed, 23 skipped
2 scenarios passed, 0 failed, 539 skipped
12 steps passed, 0 failed, 7056 skipped, 0 undefined
Took 0m8.583s

real	0m15.825s
user	0m1.906s
sys	0m0.471s
+ status=0
+ docker-compose -p gpstop_demo -f arenadata/docker-compose.yaml --env-file arenadata/.env down -v
Stopping gpstop_demo_sdw3_1 ... done
Stopping gpstop_demo_sdw1_1 ... done
Stopping gpstop_demo_sdw2_1 ... done
Stopping gpstop_demo_cdw_1  ... done
Removing gpstop_demo_sdw3_1 ... done
Removing gpstop_demo_sdw1_1 ... done
Removing gpstop_demo_sdw2_1 ... done
Removing gpstop_demo_cdw_1  ... done
Removing network gpstop_demo_default
+ [[ 0 > 0 ]]
+ exit 0
+ exits=0
+ [[ 0 > 0 ]]
```
is `locale-gen` command Ubuntu-specific while docker image based on Centos?

Reply:
Docker image building now involves glibc-common reinstallation which contains all missing locales, therefore locale-gen is not needed anymore